traitsui/testing/tester/wx/helpers.py

```diff
@@ -43,7 +43,7 @@ def key_click(widget, key, delay=0):
         else:
             wx.MilliSleep(delay)
             key_event = wx.KeyEvent(wx.wxEVT_CHAR)
-            key_event.SetUnicodeKey(KEY)
+            key_event.SetKeyCode(KEY)
```

Comment:
I think this results in another test to fail:
```
======================================================================
FAIL: test_key_click (traitsui.testing.tester.wx.tests.test_helpers.TestInteractions)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/kchoi/Work/ETS/traitsui-clone/traitsui/testing/tester/wx/tests/test_helpers.py", line 96, in test_key_click
    self.assertEqual(textbox.Value, "A")
AssertionError: 'a' != 'A'
- a
+ A
```

I think the test failure makes sense because to type "A", one needs to have the cap on or have used the Shift key. Apparently Qt disagrees and would happily insert a capitalized `"A"` in that case. So the two disagree and we should normalize it. 😂 
That's orthogonal though we should open a separate issue to solve.

Reply:
Once #1176 is merged and if we re-run the CI for this PR we should see this test failure.

Reply:
Should I modify the test to pass and comment on the discrepancy?  I can open a separate issue about it

Reply:
I realized that the tests with wx and key clicks are actually failing on Windows.... this is getting orthogonal. Let's revert this change to `SetKeyCode` and modify the tests for RangeEditor here so that it does not rely on backspaces, then open a separate issue to fix the discrepancies.